pyms/flask/services/tracer.py

```diff
@@ -71,7 +71,8 @@ def init_jaeger_tracer(self):
         metrics_config = get_conf(service=get_service_name(service="metrics"), empty_init=True)
         metrics = ""
         if metrics_config:
-            metrics = PrometheusMetricsFactory()
+            service_name = self.component_name.lower().replace("-", "_").replace(" ", "_")
+            metrics = PrometheusMetricsFactory(service_name_label=service_name)
```

Comment:
Fail the tests with `service_name_label` param :wink:

Reply:
My bad, I thought I executed them locally. I just forgot to modify the dependency in tests requirements.